cmd/envtool/envtool.go

```diff
@@ -361,6 +363,18 @@ func read(w io.Writer, paths ...string) error {
 	return nil
 }
 
+// printVersion will print out version omitting leading v
```

Comment:
The linter complains about one tiny thing: the comments should be ended with `.`

Reply:
Done!